### DISCUSSION TITLE
Stesso Genitore per più studenti

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#323] [Idee e proposte]
Stesso Genitore per più studenti

**Comment by scuolagioco** [comment]
Si, concordo con lei. Le ho inviato delle mail per un aiuto con l'aggiornamento del registro, se per piacere può rispondermi ne sarei davvero grato.. scusi per lo stress😅

**Comment by trinko** [comment]
> Stesso Genitore per più studenti

E' già gestito. 
Gli alunni devono essere inseriti con i dati dello stesso genitore (o di entrambi). I dati fondamentali che identificano lo stesso genitore sono: NOME, COGNOME, CODICE FISCALE.

Una volta così definiti, al genitore appare una schermata di scelta del profilo dell'alunno desiderato.

**Reply by manuelbarberi** [reply]
Mi porta che sono già stati inseriti quei dati quando creo altri alunni e assegno le stesse utenze ai genitori

**Reply by trinko** [reply]
> Mi porta che sono già stati inseriti quei dati quando creo altri alunni e assegno le stesse utenze ai genitori

Si, devi creare utenti genitori distinti, cioè con username diverso. Ma dovranno avere stesso NOME,COGNOME e CODICE FISCALE.
Naturalmente non sarà necessario generare le password anche per il secondo utente.

**Comment by sempliciniluigi89** [comment]
Buongiorno ,ho appena provato quanto detto , ovvero creare utenti genitori distinti , ma con stesso nome ,cognome e codice fiscale  e posso affermare che , entrando nel portale di uno dei due genitori non cambia niente, continua a dare sempre e solo informazioni relativi a un figlio .

**Reply by trinko** [reply]
> Buongiorno ,ho appena provato quanto detto , ovvero creare utenti genitori distinti , ma con stesso nome ,cognome e codice fiscale e posso affermare che , entrando nel portale di uno dei due genitori non cambia niente, continua a dare sempre e solo informazioni relativi a un figlio .

Questo è molto strano. 
Sicuro di aver effettuato il normale login dell'utente? Perché accedendo con ALIAS da amministratore è normale che non funzioni.

**Comment by sempliciniluigi89** [comment]
Si,ho provato e proprio per evitare problemi da alias ,ho provato con le
credenziali dell'utente

Il giorno lun 29 ago 2022 alle ore 19:01 Antonello Dessì <
***@***.***> ha scritto:
